packages/react/package.json

```diff
@@ -7,18 +7,23 @@
   "license": "LGPL-3.0-or-later",
   "type": "module",
   "dependencies": {
-    "@lit-labs/react": "2.1.2"
-  },
-  "peerDependencies": {
+    "@buildwithsygma/sygmaprotocol-widget": "workspace:^",
+    "@lit/react": "1.0.3",
     "react": ">=17",
     "react-dom": ">=17"
```

Comment:
shouldn't react deps be peer deps?

Reply:
I had it this way, but I wasn't able to build it in order to test it

Reply:
you should add react as both peer (range) and dev dep (exact version) :)

Reply:
done